docs/ops.html

```diff
@@ -1545,6 +1545,51 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
         <td>kafka.server:type=BrokerTopicMetrics,name=BytesRejectedPerSec,topic=([-.\w]+)</td>
         <td>Rejected byte rate per topic, due to the record batch size being greater than max.message.bytes configuration. Omitting 'topic=(...)' will yield the all-topic rate.</td>
       </tr>
+      <tr>
+        <td>The number of bytes read from remote storage per second</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=RemoteFetchBytesPerSec,topic=([-.\w]+)</td>
+        <td>Rate of bytes read from remote storage per topic. Omitting 'topic=(...)' will yield the all-topic rate</td>
+      </tr>
+      <tr>
+        <td>The number of remote storage read requests per second.</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=RemoteFetchRequestsPerSec,topic=([-.\w]+)</td>
+        <td>Rate of read requests from remote storage per topic. Omitting 'topic=(...)' will yield the all-topic rate</td>
+      </tr>
+      <tr>
+        <td>The number of remote storage read errors per second.</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=RemoteFetchErrorsPerSec,topic=([-.\w]+)</td>
+        <td>Rate of read errors from remote storage per topic. Omitting 'topic=(...)' will yield the all-topic rate</td>
+      </tr>
+      <tr>
+        <td>The number of bytes copied to remote storage per second.</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=RemoteCopyBytesPerSec,topic=([-.\w]+)</td>
+        <td>Rate of bytes copied to remote storage per topic. Omitting 'topic=(...)' will yield the all-topic rate</td>
+      </tr>
+      <tr>
+        <td>The number of remote storage write requests per second</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=RemoteCopyRequestsPerSec,topic=([-.\w]+)</td>
+        <td>Rate of write requests to remote storage per topic. Omitting 'topic=(...)' will yield the all-topic rate</td>
+      </tr>
+      <tr>
+        <td>The number of remote storage write errors per second.</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=RemoteCopyErrorsPerSec,topic=([-.\w]+)</td>
+        <td>Rate of write errors from remote storage per topic. Omitting 'topic=(...)' will yield the all-topic rate</td>
+      </tr>
+      <tr>
+        <td>Number of remote storage read tasks pending for execution</td>
+        <td>kafka.log.remote:type=RemoteLogManager,name=RemoteLogReaderTaskQueueSize</td>
```

Comment:
the "type" is incorrect here. It should be "RemoteStorageThreadPool". Can I request you to validate that these metrics are being emitted with the configuration that you specify in this review. It will help identify potential bugs.

https://github.com/apache/kafka/blob/d34d84dbef20559e68c899315a0915e9dd740cb0/storage/api/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteStorageMetrics.java#L64

Reply:
Fixed it.

Reply:
```
java -jar cmdline-jmxclient-0.10.3.jar - localhost:1099 | grep server | grep  'Remote' | grep PerSec
kafka.server:name=RemoteCopyBytesPerSec,type=BrokerTopicMetrics
kafka.server:name=RemoteFetchRequestsPerSec,type=BrokerTopicMetrics
kafka.server:name=RemoteCopyRequestsPerSec,type=BrokerTopicMetrics
kafka.server:name=RemoteFetchErrorsPerSec,type=BrokerTopicMetrics
kafka.server:name=RemoteCopyErrorsPerSec,type=BrokerTopicMetrics
kafka.server:name=RemoteFetchBytesPerSec,type=BrokerTopicMetrics

java -jar cmdline-jmxclient-0.10.3.jar - localhost:1099 | grep -i 'ThreadPool'
org.apache.kafka.storage.internals.log:name=RemoteLogReaderAvgIdlePercent,type=RemoteStorageThreadPool
org.apache.kafka.storage.internals.log:name=RemoteLogReaderTaskQueueSize,type=RemoteStorageThreadPool

java -jar cmdline-jmxclient-0.10.3.jar - localhost:1099 | grep -i 'kafka.log.remote'
kafka.log.remote:name=RemoteLogManagerTasksAvgIdlePercent,type=RemoteLogManager
```